src/styles/components/_card.scss

```
@@ -0,0 +1,55 @@
+.product-list {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.product-card {
+  text-align: center;
+  width: 24%;
+  padding: 1rem 0.1rem;
+}
+
+@media screen and (max-width: 800px) {
+  .product-card {
+    width: 50%;
+  }
+}
+
+.product-card img {
+  width: 100%;
+}
+
+.product-name {
+  width: 100%;
+  margin-top: 1rem;
+  margin-bottom: 0.5rem;
```

Fica mais limpo com o shorthand

```suggestion
  margin: 1rem 0 0.5rem;
```